.github/CODEOWNERS

```diff
@@ -1479,6 +1486,7 @@ x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts @elastic/kib
 #CC# /x-pack/plugins/security/ @elastic/kibana-security
 
 # Response Ops team
+/x-pack/test/screenshot_creation/apps/response_ops_docs @elastic/response-ops # Does this team also own x-pack/test/screenshot_creation???
```

Comment:
> /x-pack/test/screenshot_creation/apps/response_ops_docs @elastic/response-ops # Does this team also own x-pack/test/screenshot_creation???

I think it's more appropriate for the `@platform-docs` to own screenshot generation since it spans multiple content areas.

Reply:
https://github.com/elastic/kibana/pull/197468/commits/4fc988a9d85f039176685938ff95a147d140e4e2